### DISCUSSION TITLE
When will SDK 1.11 be released?

### DISCUSSION THREAD

**Discussion by Colan71** [OP-TED/eForms-SDK#827] [Q&A] [answered]
Do you have a time plan for when the SDK 1.11 including new NUTS codes will be released?

Best regards,
Asa

**Comment by YvesJo** [comment, marked as answer]
Hi,
Question was already asked (https://github.com/OP-TED/eForms-SDK/discussions/804 ) For such non-technical questions and for a more precise information, please get in touch with the helpdesk: ted@publications.europa.eu
